### DISCUSSION TITLE
Сonverting volume to tenant

### DISCUSSION THREAD

**Discussion by netscrol2016** [apache/ozone#6021] [FAQ]
Hello, please tell me, can a data volume be converted into a tenant?

**Comment by arp7** [comment]
cc @prashantpogde

**Comment by netscrol2016** [comment]
Nobody knows the answer?

**Reply by prashantpogde** [reply]
No an existing volume can not be converted to a Tenant. @smengcl was working on an enhancement for this. Also tagging @smengcl  here.

**Comment by smengcl** [comment]
Hi @netscrol2016 , we actually do have a hidden client config to force tenant creation on an existing volume. Just append `--force` to tenant creation command.

```bash
ozone tenant create existingvolname --force
```

But admins will need to understand the consequence of this. [Default tenant volume policies](https://ozone.apache.org/docs/1.3.0/feature/s3-multi-tenancy-access-control.html) will be enforced upon this existing volume, which might be undesired. This "conversion" is also not fully tested. These are some of the reasons this remains a hidden option. Also the tenant name must be exactly the same as the volume name.

**Reply by netscrol2016** [reply]
Thank you so much
